server/controllers/availability.js

```diff
@@ -0,0 +1,99 @@
+const Availability = require("../models/availability");
+const asyncHandler = require("express-async-handler");
+const Profile = require("../models/Profile");
+
+// @route POST /schedule/
+// @desc Create a schedule
+// @access Public
+exports.createSchedule = asyncHandler(async (req, res, next) => {
+  const profileId = req.profile.id;
```

Comment:
Question: Does the `req` object contain the profile id? As far as my knowledge goes, we only have access to the user id and not the profile?

Reply:
No,req obeject does not contain profile id. Made the changes to get the user id from req object and from user id get the profile id from profile model.